openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1282,19 +1282,19 @@ Cache-Control: no-store
 
 ## Client Metadata {#client_metadata}
 
-This specification defines the following new Client Metadata parameter in addition to [@!RFC7591] for Wallets acting as OAuth 2.0 Client:
+This specification defines the following new Client Metadata parameter in addition to those defined by [@!RFC7591] for Wallets acting as OAuth 2.0 Client:
 
 * `credential_offer_endpoint`: OPTIONAL. Credential Offer Endpoint of a Wallet.
 
 ### Client Metadata Retrieval {#client_metadata_retrieval}
 
-How to obtain Client Metadata is out of scope of this specification. The profiles of this specification may define static Client Metadata values.
+How to obtain Client Metadata is out of scope of this specification. Profiles of this specification MAY also define static sets of Client Metadata values to be used.
 
 If the Credential Issuer is unable to perform discovery of the Wallet's Credential Offer Endpoint, the following claimed URL is used: `openid-credential-offer://`.
 
 ## Credential Issuer Metadata {#credential_issuer_metadata}
 
-The Credential Issuer Metadata contains information on the Credential Issuer's technical capabilities, supported Credentials and (internationalized) display information.
+The Credential Issuer Metadata contains information on the Credential Issuer's capabilities, supported Credentials, and (internationalized) display information.
```

Comment:
I think `technical` is important here. it's not like any capability of the issuer can be obtained from the metadata

Reply:
I restored "technical".